static/js/portico/landing-page.js

```diff
@@ -128,7 +128,7 @@ var apps_events = function () {
         $(".info .description").text(version_info.description);
         $(".info .link").attr("href", version_info.link);
         $(".image img").attr("src", version_info.image);
-        $download_instructions.find("a").attr("href", "/help/desktop-app-install-guide#" + version);
+        $download_instructions.find("a").attr("href", "/help/desktop-app-install-guide#installing-on-" + version);
```

Comment:
I'd consider having a value in the `version_info` data structure, even though it repeats a bit, since I think it'll be easier to grep and find things, and we can also have the below still use Apple's official "macOS" spelling.

Reply:
@timabbott do you mean something like this - 
```javascript
        windows: {
            image: "/static/images/landing-page/microsoft.png",
            alt: "Windows",
            description: "Zulip for Windows is even better than Zulip on the web, with a cleaner look, tray integration, native notifications, and support for multiple Zulip accounts.",
            link: ELECTRON_APP_URL_WINDOWS,
            show_instructions: true,
            install_guide: "/help/desktop-app-install-guide#installing-on-windows"
        },
        mac: {
            image: "/static/images/landing-page/macbook.png",
            alt: "macOS",
            description: "Zulip on macOS is even better than Zulip on the web, with a cleaner look, tray integration, native notifications, and support for multiple Zulip accounts.",
            link: ELECTRON_APP_URL_MAC,
            show_instructions: true,
            install_guide: "/help/desktop-app-install-guide#installing-on-macOS"
        }

$download_instructions.find("a").attr("href", version_info.install_guide);
```

Reply:
Yep!